targets/target_persistent.c

```diff
@@ -36,17 +80,42 @@ int main(int argc, char* argv[]) {
         base_address, code_start, code_end
     };
 
+    // Configure the stalker with a Transformer and EventSink before prefetching
+    // occurs.
+    //
     GumStalkerTransformer* transformer = gum_stalker_transformer_make_from_callback(
         instr_basic_block, &instr_range, NULL);
+    GumEventSink* event_sink = gum_event_sink_make_from_callback(GUM_COMPILE | GUM_BLOCK,
+        prefetch_log, NULL, NULL);
+    gum_stalker_follow_me(stalker, transformer, event_sink);
+    gum_stalker_deactivate(stalker);
 
     static volatile char* branding __attribute__ ((used)) = (char*) "##SIG_AFL_PERSISTENT##";
     char data[1024];
     ssize_t len;
 
     afl_setup();
     afl_start_forkserver();
-    
-    gum_stalker_follow_me(stalker, transformer, NULL);
+
+    // This is written into by "prefetch_log".
+    //
+    prefetch_compiled = g_hash_table_new(NULL, NULL);
+
+    // Read in prefetch information and pass it to the Stalker before resuming
+    // execution.
+    //
+    GHashTable *compiled = g_hash_table_new(NULL, NULL);
+    prefetch_read(compile_pipes[STDIN_FILENO], compiled);
+
+    GHashTableIter iter;
+    gpointer key, value;
+    g_hash_table_iter_init(&iter, compiled);
+    while (g_hash_table_iter_next(&iter, &key, &value)) {
+        gum_stalker_prefetch(stalker, key, TRUST);
+    }
+
+    gum_stalker_activate(stalker, prefetch_activation);
```

Comment:
```suggestion
    gum_stalker_activate(stalker, box);
```

Would recommend using `box()` as the activation target – a dummy function should only be used when there's no clear target.